acceptor/settings.py

```diff
@@ -38,8 +38,8 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    'analytics.apps.AnalyticsConfig',
-    'charts.apps.ChartsConfig',
+    'edx_global_analytics.analytics.apps.AnalyticsConfig',
+    'edx_global_analytics.charts.apps.ChartsConfig',
```

Comment:
`OLGA` - OpenedX Learners Global Analytics. `edx_olga` seems tautological with `edx` at the start.
I think `edx_global_analytics` more obviously and has good combination with second `.analytics` and `.charts`.